bindep.txt

```diff
@@ -0,0 +1,41 @@
+# This file facilitates OpenStack-CI package installation
+# before the execution of any tests.
+#
+# See the following for details:
+#  - https://docs.openstack.org/infra/bindep/
+#  - https://opendev.org/opendev/bindep/
+#
+# Even if the role does not make use of this facility, it
+# is better to have this file empty, otherwise OpenStack-CI
+# will fall back to installing its default packages which
+# will potentially be detrimental to the tests executed.
+
+# The gcc compiler
+gcc
+
+# Base requirements for RPM distros
+gcc-c++           [platform:rpm]
+git               [platform:rpm]
+libffi-devel      [platform:rpm]
+openssl-devel     [platform:rpm]
+podman            [platform:rpm]
+python3-devel     [platform:rpm !platform:rhel-7 !platform:centos-7]
+PyYAML            [platform:rpm !platform:rhel-8 !platform:centos-8 !platform:rhel-9 !platform:centos-9 !platform:fedora]
+python3-pyyaml    [platform:rpm !platform:rhel-7 !platform:centos-7]
+python3-dnf       [platform:rpm !platform:rhel-7 !platform:centos-7]
+
+# RH Mechanisms
+python-rhsm-certificates                [platform:redhat]
```

Comment:
please append ' !platform:fedora'

Reply:
I removed that dependency for now - we don't use it afaik.